### PR TITLE
Rob jack tech patch 1

### PR DESCRIPTION
**Pull request by RobJackTech**
## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you for taking the time to review my pull request!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4924"></a>

### CODE REVIEW

File: data/RobJackTech.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"name": "Robert Jackson",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Full Stack Developer",
+	"avatar": "https://github.com/RobJackTech.png",
```
Comment (Panquesito7) [inline data/RobJackTech.json:6]: The `avatar` field is no longer required as of a recent update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by RobJackTech** [comment]
Thanks for taking the time to review my pull request. I have made the changes and deleted the avatar field, as recommended.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by RobJackTech** [comment]
👍 Thanks!

File: data/RobJackTech.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Robert Jackson",
+	"displayStatsPublic": true,
```
Comment (swagataroy30) [inline data/RobJackTech.json:3]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we also don't need to add `"displayStatsPublic": true` anymore. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

**Comment by himanshu1221** [comment]
> In the recent update, we also don't need to add `"displayStatsPublic": true` anymore. So, please remove that. Updated docs: https://linkfree.eddiehub.io/docs/quickstart Thank You!

agree on this +1

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by RobJackTech** [comment]
Thank you very much SaraJaoude and everyone who assisted me!

**Pull request merged**